### PR TITLE
Add (partial) Python 3 support

### PR DESCRIPTION
**Pull request by un-def**
Hi! I ported a part of this project to Python 3 (using six, with Python 2 backward compatibility). Only SMEV server part (I don't use suds-based client).
Tested with Python 3.5 + spyne 2.11.0 (I couldn't run m3-spyne-smev with newer spyne versions even under Python 2 without my Python 3-related fixes) + Django 1.11 as SMEV service backend. This part of the project seems to work fine.

### CODE REVIEW

File: src/spyne_smev/wsse/utils.py [hunk 1]
```diff
@@ -28,7 +28,7 @@
     (False, True): _xmlns.xml_c14n_wc,
 }
 
-_c14n_params = dict((v, k) for k, v in _c14n_nsmap.iteritems())
+_c14n_params = dict((v, k) for k, v in _c14n_nsmap.items())
```
Comment (chagochkin) [inline src/spyne_smev/wsse/utils.py]:
```python
_c14n_params = dict((v, k) for k, v in six.iteritems(_c14n_nsmap))
```

File: src/spyne_smev/wsse/utils.py [hunk 2]
```diff
@@ -279,7 +279,7 @@ def verify_document(document, certificate):
     if not inc_ns is None:
         inc_ns_prefixes = inc_ns.attrib["PrefixList"].split()
         inc_ns_map = dict(
-            (k, v) for k, v in document.nsmap.iteritems()
+            (k, v) for k, v in document.nsmap.items()
```
Comment (chagochkin) [inline src/spyne_smev/wsse/utils.py]: ``six.iteritems``.

File: src/spyne_smev/wsse/utils.py [hunk 3]
```diff
@@ -318,7 +318,7 @@ def verify_document(document, certificate):
...
```
Comment (chagochkin) [inline src/spyne_smev/wsse/utils.py]: ``six.iteritems()``

**Comment by un-def** [comment]
Okay, I replaced `dict.items` with `six.iteritems`.

**Review by chagochkin — APPROVED** [review]

**Comment by toshka** [comment]
Hi, please merge this PR to master!

**Comment by spumer** [comment]
Any updates?

**Comment by chagochkin** [comment]
Look at 0.2.6 version. It has Python 3 compatibility.

**Pull request closed without merging**